src/test/moment/locale_update.js

```diff
@@ -172,3 +172,24 @@ test('update existing locale', function (assert) {
     assert.equal(moment('2017-02-01').format('YYYY MMM MMMM'), '2017 FEB Februar');
     moment.updateLocale('de', null);
 });
+
+test('normalize local name', function (assert) {
+    moment.defineLocale('duration-1', null);
+    moment.defineLocale('duration-1', {
+        relativeTime: {
+            s : 'a few seconds',
+            ss : '%d Seconds'
+        }
+    });
+    moment.locale('Duration_1');
+    moment.updateLocale('Duration_1', {
+        relativeTime: {
+            s : 'A few seconds'
+        }
+    });
+    var start = moment([2007, 1, 28]);
+    moment.relativeTimeThreshold('ss', 3);
+    assert.equal(start.from(moment([2007, 1, 28]).add({s: 3}), true), 'A few seconds', 's uses child');
+    assert.equal(start.from(moment([2007, 1, 28]).add({s: 44}), true), '44 Seconds', 'ss uses base');
+    moment.relativeTimeThreshold('ss', 44);
```

Comment:
You can get the previous value and then set it, so there is no duplication (of the constant 44).

Reply:
Fixed and thanks.